### PR TITLE
change payload sent to update feature type endpoint

### PR DESCRIPTION
**Pull request by pawelzwolenskicognite**
changes after merging: https://github.com/cognitedata/coregis/pull/801

### CODE REVIEW

**Review by vincent-cognite — APPROVED** [review]
should be green when api is deployed, so approving now so that it can be merged quickly

**Comment by pawelzwolenskicognite** [comment]
@vincent-cognite is it enough that I just merge or I need to release the python package somehow?

**Comment by vincent-cognite** [comment]
> @vincent-cognite is it enough that I just merge or I need to release the python package somehow?

This is automated.
You should also update `pyproject.toml` and `CHANGELOG.md`

**Comment by pawelzwolenskicognite** [comment]
>This is automated.
You should also update pyproject.toml and CHANGELOG.md

should we wait and include the url changes in the same version bump?

**Comment by vincent-cognite** [comment]
> > This is automated.
> > You should also update pyproject.toml and CHANGELOG.md
> 
> should we wait and include the url changes in the same version bump?

smaller focused PRs are better in my opinion

**Comment by pawelzwolenskicognite** [comment]
I agree, and I don't want to merge that PR as it is. I was telling of changing version and releasing new python package (I think we can gather all these changes into one version change instead of "releasing" every small PR)

**Comment by vincent-cognite** [comment]
Feel free!

**Comment by vincent-cognite** [comment]
@pawelzwolenskicognite 
I removed the automerge since the CHANGELOG needs to be updated

File: tests/tests_integration/test_api/test_transformation_schedules.py [hunk 1]
```diff
@@ -65,6 +65,7 @@ def other_schedule(other_transformation):
     yield from schedule_from_transformation(other_transformation)
 
 
+@pytest.mark.skip(reason="This test fails several times.")
```
Comment (pawelzwolenskicognite) [inline tests/tests_integration/test_api/test_transformation_schedules.py:68]: one should not mess with @vincent-cognite :D

Comment (vincent-cognite) [reply tests/tests_integration/test_api/test_transformation_schedules.py:68]: we have a breaking change and they fail consistently

Comment (vincent-cognite) [reply tests/tests_integration/test_api/test_transformation_schedules.py:68]: this could be related to our change of course, but I fail to see why

Comment (pawelzwolenskicognite) [reply tests/tests_integration/test_api/test_transformation_schedules.py:68]: it is not due to our changes - I saw these test failing before. Looks like API is flaky

**Comment by vincent-cognite** [comment]
@silvavelosa I have to skip the transformation tests to get a successful build. We have a breaking change in the geospatial api and some client relies on us to provide an updated experimental SDK. Do you have a quick fix?

File: tests/tests_integration/test_api/test_transformation_schedules.py (hunk 1 above)
Comment (vincent-cognite) [reply tests/tests_integration/test_api/test_transformation_schedules.py:68]: @pawelzwolenskicognite 
Ideally we should test the contract, not a live running api, as you commented a few days ago . But I don't think Cognite has the capacity to dedicate a team for working on improving the SDKs

**Pull request merged**